Sources/Public/CalendarView.swift

```diff
@@ -247,7 +247,7 @@ public final class CalendarView: UIView {
     cachedAccessibilityElements = nil
     if let element = focusedAccessibilityElement as? OffScreenCalendarItemAccessibilityElement {
       UIAccessibility.post(
-        notification: .layoutChanged,
+        notification: .screenChanged,
```

Comment:
using `.screenChanged` seems to result in the focus programmatically changing faster. Using `.layoutChanged` results in a delay.

Reply:
ah good catch, this looks more correct based on the docs too